src/options/mkoptions.py

```diff
@@ -305,7 +343,15 @@ def generate_get_impl(modules):
         else:
             ret = '{{ std::stringstream s; s << options.{}.{}; return s.str(); }}'.format(
                 module.id, option.name)
-        res.append('if ({}) {}'.format(cond, ret))
+        res.append("  case OptionEnum::{}:".format(option.enum_name()))
+        res.append("  {")
+        res.append("    {};".format(ret))
+        res.append("  }")
```

Comment:
`ret` is already a single statement (it had no braces in the if)
```suggestion
        res.append("  case OptionEnum::{}: {}".format(option.enum_name(), ret))
```